docs/sdk/wallets/auth-methods/overview.md

```diff
@@ -6,9 +6,9 @@ sidebar_position: 1
 
 With Lit Protocol, you can seamlessly onboard users into web3 using familiar authentication flows like social login and passkeys. The Lit SDK simplifies the process of generating and presenting signatures derived from authentication methods that don't require users to have an existing web3 wallet. Currently, Lit Protocol supports the following non-wallet authentication methods:
```

Comment:
i think this page as a whole is repetitive. I think the 'auth-methods.md' page serves as a good enough intro, i don't think we need to include this overview page as well

Reply:
Completely agree!
Done @a1ttech :)